src/Appwrite/Auth/Auth.php

```diff
@@ -2,11 +2,31 @@
 
 namespace Appwrite\Auth;
 
+use Appwrite\Auth\Hash\BCrypt;
+use Appwrite\Auth\Hash\MD5;
+use Appwrite\Auth\Hash\PHPass;
+use Appwrite\Auth\Hash\Argon2;
+use Appwrite\Auth\Hash\SCrypt;
+use Appwrite\Auth\Hash\SCryptModified;
 use Utopia\Database\Document;
 use Utopia\Database\Validator\Authorization;
 
 class Auth
 {
+
+    const SUPPORTED_ALGOS = [
+        'argon2' => 'Argon2',
```

Comment:
Can we use camelCase or all lower?

Reply:
We use all lower-case for user-facing stuff, and we follow the same casing as algo regarding class names internally.

Do you want me to switch to camelCase for internal classes?

Reply:
iirc correctly we stick with camelCase for everything, if not, please point me to the right direction so we can review if it had any reasoning behind it.

Reply:
I was referring to `/app/config/provieres.php`. I can see API-facing keys are lower-cased, and `name` attribute is the same case as the company's official name. But then I realized that PHP class is actually just uppercased first letter.

I updated this PR to follow sane conventions - lowercase API-facing, first-upper-case class names.

Please review again, does this look good now?